conda-store-server/conda_store_server/api.py

```diff
@@ -45,6 +47,47 @@ def create_namespace(db, name: str):
     return namespace
 
 
+def update_namespace(
+    db,
+    name: str,
+    metadata_: Optional[Dict] | None = None,
```

Comment:
Let's not use `|` since this isn't backwards compatible with 3.9. Ue `typing.Union`